### PR TITLE
Adds support for Norway

### PR DESCRIPTION
**Pull request by 6185541513**
This pull request adds JVM support for Open Broker NO. This also includes moving some existing classes between 'common' and 'se' packages.

### CODE REVIEW

File: src/main/kotlin/org/openbroker/common/model/BankAccount.kt [hunk 1]
```diff
@@ -1,10 +1,10 @@
-package org.openbroker.se.model
+package org.openbroker.common.model
 
 private val clearingNumberRegex = Regex("^[1-9][0-9]{3}\$")
 private val accountNumberRegex = Regex("^[0-9]{1,11}$")
 
 /**
- * A Swedish bank account consisting of a clearing number and an
+ * A bank account consisting of a clearing number and an
```
Comment (mantono) [inline src/main/kotlin/org/openbroker/common/model/BankAccount.kt]: Is the regex above applicable for Norwegian account and clearing numbers as well? I think we should consider creating a separate BankAccount class for each country if that is the case.

File: src/main/kotlin/org/openbroker/no/privateunsecuredloan/events/Message.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.openbroker.no.privateunsecuredloan.events
+
+import org.openbroker.common.model.Reference
+
+/**
+ * An event containing information on the loan process
+ */
+
+data class Message @JvmOverloads constructor(
```
Comment (mantono) [inline src/main/kotlin/org/openbroker/no/privateunsecuredloan/events/Message.kt]: I do not think we need `@JvmOverloads` annotation if the constructor has no default values on arguments.

File: src/main/kotlin/org/openbroker/common/model/BankAccount.kt (hunk 1 above)
Comment (6185541513) [reply src/main/kotlin/org/openbroker/common/model/BankAccount.kt]: Well, yes it is applicable. But thinking closer upon this i still think you're correct. It's an object which "feels" like it's matching in this instance, but not as a rule.
I will adjust the structure and move BankAccount from 'common' to both 'no' and 'se'.

File: src/main/kotlin/org/openbroker/no/privateunsecuredloan/events/Message.kt (hunk 2 above)
Comment (6185541513) [reply src/main/kotlin/org/openbroker/no/privateunsecuredloan/events/Message.kt]: Thanks, this has been changed now.

File: src/main/kotlin/org/openbroker/no/model/BankAccount.kt [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package org.openbroker.no.model
+
+private val clearingNumberRegex = Regex("^[1-9][0-9]{3}\$")
```
Comment (Torbacka) [inline src/main/kotlin/org/openbroker/no/model/BankAccount.kt]: You dont need backslack for end of line symbol. So it should be `^[1-9][0-9]{3}$`

Comment (6185541513) [reply src/main/kotlin/org/openbroker/no/model/BankAccount.kt]: Thanks, this has been updated.

**Comment by 6185541513** [comment]
Thanks for your input guys. Updates have been made according to suggestions and discussion. This will be merged to master.

**Pull request merged**